### PR TITLE
Add the docs for the new loadbot features

### PR DESCRIPTION
**Pull request by zivkovicmilos**
# Description

This PR adds documentation for the new loadbot features.

PSDK PR:
https://github.com/0xPolygon/polygon-sdk/pull/329

### CODE REVIEW

File: docs/additional-features/stress-testing.md [hunk 1]
```diff
@@ -8,24 +8,46 @@ title: Network stress testing
 This guide assumes that:
 
 - You have a working Polygon-SDK network up and running
```
Comment (Kourin1996) [inline docs/additional-features/stress-testing.md:10]: `Polygon SDK` is more proper to be used?

File: docs/additional-features/stress-testing.md [hunk 2]
```diff
@@ -8,24 +8,46 @@ title: Network stress testing
...
 
-e.g. to export the variable :
+## Overview
+
+The Polygon SDK loadbot is a helper that is meant to stress test the Polygon SDK cluster.
+
+Currently, it supports 2 modes:
+
+- `Transfer` - mode that does stress testing using fund-transfer transactions. **[Default]**.
+- `Deploy` - mode that deploys specified smart contracts with each transcation.
```
Comment (Kourin1996) [inline docs/additional-features/stress-testing.md:20]: I think `transfer` and `deploy` (which are same values to ones used in command) are better in order not to make users confused. How do you think?

File: docs/additional-features/stress-testing.md [hunk 3]
```diff
@@ -8,24 +8,46 @@ title: Network stress testing
...
+
+### Transfer Mode
+
+The transfer mode assumes that there is a sender account that has initial funds to conduct the loadbot run.
+
+The sender's account address and private key need to be specified within the shell that's running the loadbot command.
```
Comment (Kourin1996) [inline docs/additional-features/stress-testing.md:26]: I think `setting in environment variables` is easy to understand.

**Review by Kourin1996 — APPROVED** [review]
Looks good to me. I've left small suggestions. Please check and fix if you need. Thanks

File: docs/additional-features/stress-testing.md (hunk 1 above)
Comment (zivkovicmilos) [reply docs/additional-features/stress-testing.md]: Fixed 👍 

74f7726

File: docs/additional-features/stress-testing.md (hunk 2 above)
Comment (zivkovicmilos) [reply docs/additional-features/stress-testing.md]: You're right 👍 

Fixed:

230c12e

File: docs/additional-features/stress-testing.md (hunk 3 above)
Comment (zivkovicmilos) [reply docs/additional-features/stress-testing.md]: It does sound more clear 🙂 

Fixed 👍 

76c048d

**Pull request merged**